workflows/mobilomeannotation.nf

```diff
@@ -81,7 +84,25 @@ workflow MOBILOMEANNOTATION {
         }
     }
 
+    user_proteins_ch
```

Comment:
This works just fine but I think there is a better approach (for a future refactor!).
Using gzip in python itself or in the processes that need to handle compressed files (handling compressed files in scripts/tools is better) tends to improve the overall performance. Using gzip as a process means that a job needs to be scheduled and executed by slurm, then files have to written to disk (the uncompressed one) and symlinks created ... which is mostly just overhead.

Reply:
Update: bedtools container have gzip. I will update the pipeline to handle compressed inputs in modules and python scripts